pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -33,6 +34,153 @@ const (
 	AnnImportDiskId = "cdi.kubevirt.io/storage.import.volumeId"
 )
 
+// OS types
+const (
+	Linux   = "linux"
+	Windows = "windows"
+)
+
+// Secure boot options
+const (
+	Required = "required"
+	Disabled = "disabled"
+	Optional = "optional"
+)
+
+// Machine types
+const (
+	PC  = "pc"
+	Q35 = "q35"
+)
+
+// Firmware types
+const (
+	BIOS = "bios"
+	EFI  = "uefi"
+)
+
+// CPU Policies
+const (
+	CPUPolicyDedicated = "dedicated"
+	CPUPolicyShared    = "shared"
+)
+
+// CPU Thread policies
+const (
+	CPUThreadPolicyPrefer  = "prefer"
+	CPUThreadPolicyIsolate = "isolate"
+	CPUThreadPolicyRequire = "require"
+)
+
+// Bus types
+const (
+	VirtioSCSI = "virtio_scsi"
+	VirtIO     = "virtio"
+	SATA       = "sata"
+	SCSI       = "scsi"
+	IDE        = "ide"
```

Comment:
interesting, if that is supported in openstack (which makes sense) then we need to introduce the same logic we've added for imports from ovirt that convert such vms to q35

Reply:
ok, filed https://github.com/kubev2v/forklift/issues/257 for this